extra-database/mariadb/autobuild/defines

```diff
@@ -34,10 +34,9 @@ CMAKE_AFTER="-DCMAKE_AR=/usr/bin/gcc-ar \
              -DWITH_SSL=system \
              -DWITH_PCRE=bundled \
              -DWITH_LIBWRAP=OFF \
-             -DWITH_JEMALLOC=ON \
+             -DWITH_JEMALLOC=yes \
```

Comment:
Huh?

Reply:
The CMakeLists seems to be accepting yes/no

Reply:
see https://github.com/MariaDB/server/blob/ddcb242b3cd2e95bcf4c950bffaafc0775db45ee/cmake/jemalloc.cmake#L4

Reply:
I see, that's all good. Just making sure.